### PR TITLE
CloudFlare proxy is enabled. Allow only CloudFlare IP's.

### PR DESCRIPTION
**Pull request by artsyjian**
To enable CloudFlare proxying. Motivated by:

https://artsyproduct.atlassian.net/browse/PLATFORM-2537
https://artsy.slack.com/archives/CA8SANW3W/p1593546460444200

I have enabled Proxy feature via CF UI. So requests to `artsy.net` is now being proxied by CF. This PR is to lock down the LB to allow only CF IP's.

Screenshot from 2020-06-30 17-48-23

### CODE REVIEW

File: hokusai/production.yml [hunk 1]
```diff
@@ -87,6 +87,7 @@ metadata:
     service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
     service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: "true"
     service.beta.kubernetes.io/aws-load-balancer-connection-draining-enabled: "true"
+    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "5"
```
Comment (artsyjian) [inline hokusai/production.yml:90]: To true up to template: https://github.com/artsy/artsy-hokusai-templates/blob/2070773efe4bfd89ba3f4ef82eb02ac20f3a21f6/nodejs/hokusai/staging.yml.j2#L146

File: hokusai/production.yml [hunk 2]
```diff
@@ -103,3 +104,18 @@ spec:
     component: web
   sessionAffinity: None
   type: LoadBalancer
+  loadBalancerSourceRanges:
```
Comment (izakp) [inline hokusai/production.yml:107]: Just added a new hokusai template config var `cloudflareIpSourceRanges` - if you want to use it this should work
```
  loadBalancerSourceRanges:
{%- for cidr_block in cloudflareIpSourceRanges %}
    - {{ cidr_block }}
{%- endfor %}
```

**Comment by izakp** [comment]
Not sure why the deploy-staging job triggered on PR...

File: hokusai/staging.yml [hunk 3]
```diff
@@ -103,3 +104,17 @@ spec:
     component: web
   sessionAffinity: None
   type: LoadBalancer
+    - 173.245.48.0/20
```
Comment (izakp) [inline hokusai/staging.yml:107]: missing line `loadBalancerSourceRanges:`

Deploy fails with:
```
The Service "artsy-wwwify-web" is invalid: spec.type: Unsupported value: "LoadBalancer - 173.245.48.0/20 - 103.21.244.0/22 - 103.22.200.0/22 - 103.31.4.0/22 - 141.101.64.0/18 - 108.162.192.0/18 - 190.93.240.0/20 - 188.114.96.0/20 - 197.234.240.0/22 - 198.41.128.0/17 - 162.158.0.0/15 - 104.16.0.0/12 - 172.64.0.0/13 - 131.0.72.0/22": supported values: "ClusterIP", "ExternalName", "LoadBalancer", "NodePort"
```

Comment (artsyjian) [reply hokusai/staging.yml:107]: Thanks @izakp .

File: hokusai/production.yml (hunk 2 above)
Comment (artsyjian) [reply hokusai/production.yml:107]: cool.

**Pull request merged**